### PR TITLE
T3 added

### PR DESCRIPTION
**Pull request by pleiadae**
TestMenuPage class was added
New methods and attributes were added to MenuPage
TestMenuPage was added to testng.xml

### CODE REVIEW

File: src/main/java/com/endava/pages/MenuPage.java [hunk 1]
```diff
@@ -20,4 +23,37 @@ public MenuPage(WebDriver driver) {
...
+		return investorsAboutUs;
+	}
+
+	/**
+	 * finds the INVESTORS element in burger menu and clicks on it
+	 * 
+	 * @author jelena.v.corak@gmail.com
+	 */
+	public void clickOnInvestors() {
+		WebElement investors = driver.findElement(burgerMenuOptionInvestors);
```
Comment (endavamarko) [inline src/main/java/com/endava/pages/MenuPage.java]: We should assert is element present and print message if element is not present, and than to click. If we don't check this and method fails, we don't know is it not present or not clickable. Example with pseudo-code:
 log.debug("opis metode");
    isElementPresent(locator,Time.LONG);
    locator.click();
There are many ways to do that:
Most simple is check is webelements array empty 
Other methods:
public static void waitForElementToAppear(Driver driver, By selector, long timeOutInSeconds, String timeOutMessage) {
    try {
      WebDriverWait wait = new WebDriverWait(driver, timeOutInSeconds);
      wait.until(ExpectedConditions.visibilityOfElementLocated(selector));
    } catch (TimeoutException e) {
      throw new IllegalStateException(timeOutMessage);
    }
  }
Method with displayed()
 public boolean isElementPresent(WebElement locator) {
    try {
      return locator.isDisplayed();
    }
    catch (final Exception e) {
      return false;
    }
  }

File: src/test/java/com/endava/TestMenuPage.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+	 *
+	 * Test validates that string "ABOUT US" is visible
+	 * on the "INVESTORS" page in burger menu
+	 */
+	@Test
+	public void isAboutUsVisible() {
+		
+		homePage.open();		
+		new WebDriverWait(homePage.driver, 5)
+				.until(ExpectedConditions.visibilityOfElementLocated(homePage.getContactButtons()));
```
Comment (endavamarko) [inline src/test/java/com/endava/TestMenuPage.java]: WeDriverWait ne treba u test metodi vec staviti u page klasu Utils metodu koja se zove iz test metode sa parametrima

File: src/main/java/com/endava/pages/MenuPage.java [hunk 3]
```diff
@@ -11,6 +14,8 @@
 public class MenuPage extends BasePage {
 
 	private By navigationList = By.className("navigation");
+	private By burgerMenuOptionInvestors = By.cssSelector(".navList > li:nth-child(5) > a:nth-child(1)");
```
Comment (BoskoNikolic) [inline src/main/java/com/endava/pages/MenuPage.java]: koristi xpath ovde

Comment (BoskoNikolic) [inline src/main/java/com/endava/pages/MenuPage.java]: nazovi recimo investorsMenuItem  ovo "burger" je vise nas lokalni naziv :D

File: src/main/java/com/endava/pages/MenuPage.java [hunk 4]
```diff
@@ -20,4 +25,41 @@ public MenuPage(WebDriver driver) {
...
+	}
+
+	public By getInvestorsAboutUs() {
+		return investorsAboutUs;
+	}
+
+	/**
+	 * finds the INVESTORS element in burger menu and clicks on it
+	 * 
+	 * @author jelena.v.corak@gmail.com
```
Comment (BoskoNikolic) [inline src/main/java/com/endava/pages/MenuPage.java]: ispravi u jelena.corak nemoj da ostavljas mail. Ispravi ovo svugde gde si stavila

File: src/main/java/com/endava/pages/MenuPage.java [hunk 5]
```diff
@@ -20,4 +25,41 @@ public MenuPage(WebDriver driver) {
...
+	 */
+	public void clickOnInvestors() {
+		WebElement investors = driver.findElement(burgerMenuOptionInvestors);
+		if (Utils.isElementPresent(investors)) {
+			investors.click();
+			log.debug("clickOnInvestors(): Clicked on the element INVESTORS.");
+		} else {
+			log.debug("clickOnInvestors(): Element INVESTORS is not present.");
```
Comment (BoskoNikolic) [inline src/main/java/com/endava/pages/MenuPage.java]: fali ti nesto nakon ovoga, da oboris test recimo

File: src/main/java/com/endava/util/Utils.java [hunk 6]
```diff
@@ -2,35 +2,58 @@
...
+	 * @author jelena.v.corak@gmail.com
+	 * @author Vladimir Krekic
+	 *
+	 * @param WebElement
+	 * @return boolean
+	 * 
+	 */
+	public static boolean isElementPresent(WebElement element) {
+		try {
+			return element.isDisplayed();
```
Comment (BoskoNikolic) [inline src/main/java/com/endava/util/Utils.java]: isDisplayed vraca true/fals ne treba nikakav try/catch blok oko njega

**Pull request closed without merging**